lib/passenger.rb

```diff
@@ -0,0 +1,22 @@
+class Passenger
+  attr_reader :name,
+              :age,
+              :drive
```

Comment:
```suggestion
              :driver
```
Since the attribute is `@driver` this should also be `driver`